### ISSUE TITLE
AVT 1 - Expandable Tile: DAP violation the form control is missing an associated label

### ISSUE THREAD

**Issue by snidersd** [IBM/carbon-components-react#2242]
## Environment
macOS Mojave version 10.14.2
Chrome version 72.0.3626.96 (Official Build) (64-bit)

## Detailed Description
Run DAP with Feb. 2019 ruleset on the Expandable Tile component
See screenshot below for violation:
<img width="1060" alt="Screen Shot 2019-04-18 at 3 34 21 PM" src="https://user-images.githubusercontent.com/21676914/56389730-e3e4a000-61ef-11e9-80e8-ad7d71b376a8.png">
